README.md

```diff
@@ -1159,9 +1159,20 @@ Use the following setting to optimize the audio bitrate of content share audio f
 meetingSession.audioVideo.setContentAudioProfile(AudioProfile.fullbandMusicMono());
 ```
 
+**Use case 34.** Set the audio quality of content share audio to optimize for music with stereo channel:
```

Comment:
Is there any plan to write a more in-depth guide on audio? For now, can we add when they want to enable this?
The use case here is for content audio but for testing we just test audio so seem a bit weird.

Reply:
Updated this section a bit.